modelplace_api/visualization.py

```diff
@@ -286,14 +317,14 @@ def draw_pose_estimation_result(
                     image_with_skeletons,
                     (int(link.joint_a.x), int(link.joint_a.y)),
                     (int(link.joint_b.x), int(link.joint_b.y)),
-                    RGB_COLORS[-1][::-1],
+                    RGB_COLORS[-1][2::-1],
```

Comment:
Let's rename `RGB_COLORS` to `RGBA_COLORS` for a better understanding of why we do this way